license-scan/Cargo.toml

```diff
@@ -9,14 +9,14 @@ publish = false
 [dependencies]
 anyhow = "1"
 askalono = "0.4"
-cargo_metadata = "0.14"
+cargo_metadata = "0.18"
 ignore = "0.4"
 lazy_static = "1"
 semver = { version = "1", features = ["serde"] }
 serde = { version = "1", features = ["derive"] }
 spdx = "0.3"
```

Comment:
I skipped bumping `spdx` because in `0.6.2` they added a directory full of text of licenses and exemptions that were being caught by the license scanner. Options are just bump to `0.6.1` (which is still rather old), or enhance the license scanner (#171) before jumping to the latest.